sunpy/net/vso/attrs.py

```diff
@@ -536,34 +282,37 @@ def _create(wlk, root, api):
 # AttrAnd and AttrOr obviously are - in the HEK module). If we defined the
 # filter method as a member of the attribute classes, we could only filter
 # one type of data (that is, VSO data).
-filter_results = MultiMethod(lambda *a, **kw: (a[0], ))
+
+@_singledispatch
+def _filter_results(*args, **kwargs):
```

Comment:
Well we don't have any tests for this, so hopefully it's not broken :see_no_evil:

Reply:
You add one?

Reply:
I would rather fold it into https://github.com/sunpy/sunpy/issues/3733 really.

Reply:
Fine by me.